docs/core/tools/sdk-errors/netsdk1141.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "NETSDK1141: Unable to resolve the .NET SDK version as specified in the global.json"
+description: Learn about .NET SDK error NETSDK1141, which occurs when the SDK version could not be resolved in global.json.
+ms.topic: error-reference
+ms.date: 04/16/2021
+f1_keywords:
+- NETSDK1141
+---
+# NETSDK1141: Unable to resolve the .NET SDK version as specified in the global.json
+
+**This article applies to:** ✔️ .NET Core 2.1.100 SDK and later versions
```

Comment:
This error looks much more recent than 2.1. It was added in https://github.com/dotnet/sdk/pull/12633.

Reply:
@Youssef1313 @tdykstra OK, so it was merged August 2020. Does that mean it was only added for .NET 5 ?

Reply:
The [issue](https://github.com/dotnet/sdk/issues/11238) says it's for the 5.0.1xx milestone, so I would change the Applies-to line, though I think these Applies-to lines on error messages are superfluous -- if you get the error message and look it up, it definitely applies to you.

```suggestion
**This article applies to:** ✔️ .NET Core 5.0 SDK and later versions
```